Apps/W1/HybridGP/app/src/Migration/Vendors/GPVendorMigrator.codeunit.al

```diff
@@ -376,4 +376,100 @@ codeunit 4022 "GP Vendor Migrator"
         HelperFunctions.UpdateFieldValue(RecordVariant, GPVendorTransactions.FieldNo(PYMTRMID), JToken.AsObject(), 'PYMTRMID');
         HelperFunctions.UpdateFieldWithValue(RecordVariant, GPVendorTransactions.FieldNo(GLDocNo), DocumentNo);
     end;
+
+    procedure MigrateVendorEFTBankAccounts()
+    var
+        GPSY06000Table: Record GPSY06000;
+        VendorTable: Record Vendor;
+        VendorBankAccountTable: Record "Vendor Bank Account";
```

Comment:
Minor - we do not suffix the records with table in the code. It is the existing coding style.